Makefile.js

```diff
@@ -695,7 +695,7 @@ target.gensite = function(prereleaseVersion) {
                 "---",
                 `title: ${title}`,
                 "layout: doc",
-                `edit_link: https://github.com/eslint/eslint/edit/master/${filePath}`,
+                `edit_link: https://github.com/eslint/eslint/edit/HEAD/${filePath}`,
```

Comment:
It seems that this one doesn't work.

I tried with https://eslint.org/docs/user-guide/getting-started page. "Edit this page" link at the bottom currently points to:

https://github.com/eslint/eslint/edit/master/docs/user-guide/getting-started.md

That works, but if we change "master" to "HEAD", this returns 404:

https://github.com/eslint/eslint/edit/HEAD/docs/user-guide/getting-started.md

Reply:
Maybe, it is because we can use HEAD only for reference to a file to the latest commit, but cannot try making edits to that same commit (not sure just my guess)
Should I replace it with `main`?

Reply:
Ah, that’s unfortunate, GitHub must have missed one.

Reply:
Yes, let’s switch to main for now.